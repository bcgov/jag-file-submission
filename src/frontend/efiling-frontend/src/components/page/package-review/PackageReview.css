.align-icon {
  margin-left: 10px;
  vertical-align: text-top;
}
.file-href {
  cursor: pointer;
  color: #1a5a96;
  text-decoration: underline;
}
