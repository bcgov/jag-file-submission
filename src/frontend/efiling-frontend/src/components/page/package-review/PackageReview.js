import React, { useEffect, useState } from "react";
import moment from "moment";
import PropTypes from "prop-types";
import { Header, Footer, Button, Table, Alert } from "shared-components";
import { MdCancel } from "react-icons/md";
import axios from "axios";
import { propTypes } from "../../../types/propTypes";

export default function PackageReview({ page: { header, packageId } }) {
  const [error, setError] = useState(false);
  const [packageDetails, setPackageDetails] = useState([
    {
      name: "Package Number:",
      value: "",
      isNameBold: false,
      isValueBold: true,
    },
    { name: "Submitted By:", value: "", isNameBold: false, isValueBold: true },
    {
      name: "Submitted Date:",
      value: "",
      isNameBold: false,
      isValueBold: true,
    },
  ]);
  const [courtFileDetails, setCourtFileDetails] = useState([
    {
      name: "Court File Number:",
      value: "",
      isNameBold: false,
      isValueBold: true,
    },
    { name: "Submitted To:", value: "", isNameBold: false, isValueBold: true },
    {
      name: "Filing Comments:",
      value: "",
      isNameBold: false,
      isValueBold: true,
    },
  ]);

  useEffect(() => {
    axios
      .get(`filingpackage/${packageId}`)
      .then((response) => {
        try {
          const packageNo = response.data.packageNumber || "";
          let submittedBy = "";
          if (response.data.submittedBy.firstName) {
            if (response.data.submittedBy.lastName) {
              submittedBy = `${response.data.submittedBy.firstName} ${response.data.submittedBy.lastName}`;
            }
          }
          let submittedDt = "";
          if (response.data.submittedDate) {
            submittedDt = moment(response.data.submittedDate).format(
              "DD-MMM-YYYY HH:MM"
            );
          }
          const fileNumber = response.data.court.fileNumber || "";
          const submittedTo = response.data.court.location || "";
          const filingComments = response.data.filingComments || "";
          setPackageDetails([
            {
              name: "Package Number:",
              value: `${packageNo}`,
              isNameBold: false,
              isValueBold: true,
            },
            {
              name: "Submitted By:",
              value: submittedBy,
              isNameBold: false,
              isValueBold: true,
            },
            {
              name: "Submitted Date:",
              value: submittedDt,
              isNameBold: false,
              isValueBold: true,
            },
          ]);
          setCourtFileDetails([
            {
              name: "Court File Number:",
              value: `${fileNumber}`,
              isNameBold: false,
              isValueBold: true,
            },
            {
              name: "Submitted To:",
              value: `${submittedTo}`,
              isNameBold: false,
              isValueBold: true,
            },
            {
              name: "Filing Comments:",
              value: `${filingComments}`,
              isNameBold: false,
              isValueBold: true,
            },
          ]);
        } catch (err) {
          setError(true);
        }
      })
      .catch(() => {
        setError(true);
      });
  }, []);

  return (
    <main>
      <Header header={header} />
      <div className="page">
        <div className="content col-md-12">
          <h1>View Submitted Package</h1>
          {error && (
            <div className="col-md-12">
              <Alert
                icon={<MdCancel size={24} />}
                type="error"
                styling="bcgov-error-background"
                element="There was a problem with your request."
              />
            </div>
          )}
          <br />
          <h2>Package Details</h2>
          <br />
          <div className="row">
            <div className="col-sm-12 col-lg-6">
              <Table elements={packageDetails} />
            </div>
            <div className="col-sm-12 col-lg-6">
              <Table elements={courtFileDetails} />
            </div>
          </div>
          <br />
          <br />
          <section className="buttons pt-2">
            <Button
              label="Cancel and Return to Parent App"
              onClick={() => window.open("http://google.com", "_self")}
              styling="normal-white btn"
            />
          </section>
        </div>
      </div>
      <Footer />
    </main>
  );
}

PackageReview.propTypes = {
  page: PropTypes.shape({
    header: propTypes.header,
    packageId: PropTypes.string.isRequired,
  }).isRequired,
};
