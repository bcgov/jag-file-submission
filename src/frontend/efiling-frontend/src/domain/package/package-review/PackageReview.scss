.ct-package-review {
  .align-icon {
    margin-left: 10px;
    vertical-align: text-top;
  }
  .registry-align {
    margin-top: 15%;
  }
  .tabContent {
    border: 1px solid grey;
    padding: 10px;
    margin-top: 10px;
    white-space: pre-line;
  }
  .note.row, .buttons.row {
    margin: 0px 0px 10px 0px;
  }

  /* REMOVE-ME when replacing bootstrap Tabs for common-components Tabs */
  :focus {
    outline: none;
  }
  .fw-bold {
    font-weight: bold;
  }

  @media (max-width: 768px) {
    .download-alert {
      position: fixed !important;
      width: 85%;
      top: 100px;
    }
  }
}
