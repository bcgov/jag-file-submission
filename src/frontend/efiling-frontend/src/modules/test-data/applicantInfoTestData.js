const applicantInfo = {
  bceid: "bobross42",
  firstName: "Bob",
  lastName: "Ross",
  email: "bob.ross@example.com",
};

export function getApplicantInfo() {
  return applicantInfo;
}
