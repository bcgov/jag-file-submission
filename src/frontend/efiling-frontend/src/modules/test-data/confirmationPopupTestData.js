/* eslint-disable react/jsx-one-expression-per-line */
import React from "react";

const onButtonClick = () => {};

const mainButton = {
  label: "Cancel",
  styling: "bcgov-normal-white btn",
  onClick: onButtonClick,
};

const confirmButton = {
  label: "Yes, cancel",
  styling: "bcgov-normal-blue btn bcgov-consistent-width",
  onClick: onButtonClick,
};

const cancelButton = {
  label: "No, dont cancel",
  styling: "bcgov-normal-white btn bcgov-consistent-width",
  onClick: onButtonClick,
};

const body = () => (
  <>
    <p>Your files will not be submitted.</p>
    <p>
      You will be returned to:
      <br />
      <b>Original</b> website
    </p>
  </>
);

export function getTestData() {
  const modal = {
    show: false,
    title: "title",
    body,
  };

  return {
    modal,
    mainButton,
    confirmButton,
    cancelButton,
  };
}
