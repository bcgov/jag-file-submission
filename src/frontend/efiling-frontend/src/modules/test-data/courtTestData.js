const court = {
  locationDescription: "Court location",
  fileNumber: "Court file number",
  levelDescription: "Level",
  classDescription: "Class",
  level: "P",
  courtClass: "F",
  location: "Kelowna Law Courts",
};

export function getCourtData() {
  return court;
}
