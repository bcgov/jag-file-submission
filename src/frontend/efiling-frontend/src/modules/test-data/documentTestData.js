const documents = [
  {
    documentProperties: {
      name: "file name 1",
      type: "file type",
    },
    description: "file description 1",
    statutoryFeeAmount: 40,
    isAmendment: null,
    isSupremeCourtScheduling: null,
  },
  {
    documentProperties: {
      name: "file name 2",
      type: "file type",
    },
    description: "file description 2",
    statutoryFeeAmount: 0,
    isAmendment: false,
    isSupremeCourtScheduling: true,
  },
];

export function getDocumentsData() {
  return documents;
}
