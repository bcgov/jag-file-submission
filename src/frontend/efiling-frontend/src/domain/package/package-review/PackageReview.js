/* eslint-disable react/jsx-one-expression-per-line */

import React, { useEffect, useState } from "react";
import moment from "moment";
import Tabs from "react-bootstrap/Tabs"; /* TODO: replace with shared-components */
import Tab from "react-bootstrap/Tab"; /* TODO: replace with shared-components */
import { Button, Table, Alert } from "shared-components";
import { BsEyeFill } from "react-icons/bs";
import { MdCancel } from "react-icons/md";
import { useParams } from "react-router-dom";
import { errorRedirect } from "../../../modules/helpers/errorRedirect";
import {
  getFilingPackage,
  downloadSubmissionSheet,
} from "./PackageReviewService";
import { noop } from "../../../modules/helpers/mockHelper";

import "./PackageReview.scss";
import DocumentList from "./DocumentList";
import ParyList from "./PartyList";
import PaymentList from "./PaymentList";

export default function PackageReview() {
  const { packageId } = useParams();
  const [error, setError] = useState(false);
  const [packageDetails, setPackageDetails] = useState([
    { name: "Submitted By:", value: "", isNameBold: false, isValueBold: true },
    {
      name: "Submitted Date:",
      value: "",
      isNameBold: false,
      isValueBold: true,
    },
    { name: "Submitted To:", value: "", isNameBold: false, isValueBold: true },
  ]);
  const [courtFileDetails, setCourtFileDetails] = useState([
    {
      name: "Package Number:",
      value: "",
      isNameBold: false,
      isValueBold: true,
    },
    {
      name: "Court File Number:",
      value: "",
      isNameBold: false,
      isValueBold: true,
    },
  ]);
  const [filingComments, setFilingComments] = useState("");
  const [reloadTrigger, setReloadTrigger] = useState(false);
  const [documents, setDocuments] = useState([]);
  const [parties, setParties] = useState([]);
  const [payments, setPayments] = useState([]);

  useEffect(() => {
    getFilingPackage(packageId)
      .then((response) => {
        try {
          const packageNo = response.data.packageNumber || "";
          let submittedBy = "";
          if (response.data.submittedBy.firstName) {
            if (response.data.submittedBy.lastName) {
              submittedBy = `${response.data.submittedBy.firstName} ${response.data.submittedBy.lastName}`;
            }
          }
          let submittedDt = "";
          if (response.data.submittedDate) {
            submittedDt = moment(response.data.submittedDate).format(
              "DD-MMM-YYYY HH:mm"
            );
          }
          const fileNumber = response.data.court.fileNumber || "";
          const submittedTo = response.data.court.location || "";
          setPackageDetails([
            {
              name: "Submitted By:",
              value: submittedBy,
              isNameBold: false,
              isValueBold: true,
            },
            {
              name: "Submitted Date:",
              value: submittedDt,
              isNameBold: false,
              isValueBold: true,
            },
            {
              name: "Submitted To:",
              value: `${submittedTo}`,
              isNameBold: false,
              isValueBold: true,
            },
          ]);
          setCourtFileDetails([
            {
              name: "Package Number:",
              value: `${packageNo}`,
              isNameBold: false,
              isValueBold: true,
            },
            {
              name: "Court File Number:",
              value: `${fileNumber}`,
              isNameBold: false,
              isValueBold: true,
            },
          ]);
          setFilingComments(response.data.filingComments);
          setDocuments(response.data.documents);
          setParties(response.data.parties);
          setPayments(response.data.payments);
        } catch (err) {
          setError(true);
        }
      })
      .catch(() => {
        setError(true);
      });
  }, [packageId, reloadTrigger]);

  /** Whenever this function is called, it'll trigger a reload of the document list. */
  function reloadDocumentList() {
    setReloadTrigger(!reloadTrigger);
    noop();
  }

  function handleClick() {
    downloadSubmissionSheet(packageId).catch((err) => {
      errorRedirect(sessionStorage.getItem("errorUrl"), err);
    });
  }

  function handleKeyDown(e) {
    if (e && e.keyCode === 13) {
      downloadSubmissionSheet(packageId).catch((err) => {
        errorRedirect(sessionStorage.getItem("errorUrl"), err);
      });
    }
  }

  return (
    <>
      <div className="ct-package-review page">
        <div className="content col-md-8">
          <h1>View Recently Submitted Package # {packageId}</h1>
          {error && (
            <div className="col-md-8">
              <Alert
                icon={<MdCancel size={24} />}
                type="error"
                styling="bcgov-error-background"
                element="There was a problem with your request."
              />
            </div>
          )}
          <br />
          <div className="row">
            <h2 className="col-sm-6">Package Details</h2>
            <div className="col-sm-6 text-sm-right mt-3 mt-sm-0">
              <span
                role="button"
                tabIndex={0}
                className="file-href"
                onClick={handleClick}
                onKeyDown={handleKeyDown}
              >
                Print Submission Sheet
              </span>
              <BsEyeFill size="24" color="#7F7F7F" className="align-icon" />
            </div>
          </div>
          <br />
          <div className="row" data-test-id="detailsTable">
            <div className="col-sm-12 col-lg-6">
              <Table id="packageDetails" elements={packageDetails} />
            </div>
            <div className="col-sm-12 col-lg-6">
              <Table elements={courtFileDetails} />
            </div>
          </div>
          <br />
          <Tabs defaultActiveKey="documents" id="uncontrolled-tab">
            <Tab eventKey="documents" title="Documents">
              <br />
              <DocumentList
                packageId={packageId}
                documents={documents}
                reloadDocumentList={reloadDocumentList}
              />
            </Tab>
            <Tab eventKey="parties" title="Parties">
              <br />
              <ParyList parties={parties} />
            </Tab>
            <Tab eventKey="payment" title="Payment Status">
              <br />
              <PaymentList payments={payments} />
            </Tab>
            <Tab eventKey="comments" title="Filing Comments">
              <br />
              <h4>Filing Comments</h4>
              <div id="filingComments" className="tabContent">
                {filingComments}
              </div>
            </Tab>
          </Tabs>
          <br />
          <section className="buttons pt-2">
            <Button
              label="Cancel and Return to Parent App"
              onClick={() => window.open("http://google.com", "_self")}
              styling="normal-white btn"
            />
          </section>
        </div>
      </div>
    </>
  );
}
