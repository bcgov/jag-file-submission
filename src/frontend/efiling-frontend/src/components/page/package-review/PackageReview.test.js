import React from "react";
import axios from "axios";
import FileSaver from "file-saver";
import { createMemoryHistory } from "history";
import { render, waitFor, fireEvent, getByText } from "@testing-library/react";

import MockAdapter from "axios-mock-adapter";
import moment from "moment-timezone";
import PackageReview from "./PackageReview";
import { getCourtData } from "../../../modules/test-data/courtTestData";

// hack fix to force GitHub to run tests in BC timezone
moment.tz.setDefault("America/Vancouver");

describe("PackageReview Component", () => {
  const header = {
    name: "eFiling Frontend",
    history: createMemoryHistory(),
  };
  const packageId = "1";
  const courtData = getCourtData();
  const submittedDate = new Date("2021-01-14T18:57:43.602Z").toISOString();
  const submittedBy = { firstName: "Han", lastName: "Solo" };
  const filingComments =
    "Lorem ipsum dolor sit amet.<script>alert('Hi');</script>\n\nDuis aute irure dolor.";

  const page = {
    header,
    packageId,
  };

  FileSaver.saveAs = jest.fn();

  let mock;
  beforeEach(() => {
    mock = new MockAdapter(axios);
    window.open = jest.fn();
  });

  const apiRequest = `/filingpackage/${packageId}`;

  test("Matches the snapshot", async () => {
    mock.onGet(apiRequest).reply(200, {
      packageNumber: packageId,
      court: courtData,
      submittedBy,
      submittedDate,
      filingComments,
    });

    const { asFragment } = render(<PackageReview page={page} />);
    await waitFor(() => {});

    expect(asFragment()).toMatchSnapshot();

    // filingComments should have properly escaped html characters
    const fc = asFragment().querySelector("#filingComments");
    expect(fc.innerHTML).toEqual(
      "Lorem ipsum dolor sit amet.&lt;script&gt;alert('Hi');&lt;/script&gt;\n\nDuis aute irure dolor."
    );
  });

  test("Clicking cancel takes user back to parent app", async () => {
    const { container } = render(<PackageReview page={page} />);
    await waitFor(() => {});

    fireEvent.click(getByText(container, "Cancel and Return to Parent App"));

    expect(window.open).toHaveBeenCalledWith("http://google.com", "_self");
  });

  test("Api called successfully when page loads with valid packageId", async () => {
    window.open = jest.fn();
    mock.onGet(apiRequest).reply(200, {
      court: courtData,
      submittedBy,
      submittedDate,
    });

    const spy = jest.spyOn(axios, "get");

    render(<PackageReview page={page} />);
    await waitFor(() => {});

    expect(spy).toHaveBeenCalled();
  });

  test("Api called, empty response data", async () => {
    window.open = jest.fn();
    mock.onGet(apiRequest).reply(200, {
      court: {},
      submittedBy: {},
      submittedDate: "",
    });

    const spy = jest.spyOn(axios, "get");

    render(<PackageReview page={page} />);
    await waitFor(() => {});

    expect(spy).toHaveBeenCalled();
  });

  test("Api called, invalid submittedBy", async () => {
    window.open = jest.fn();
    mock.onGet(apiRequest).reply(200, {
      court: courtData,
      submittedBy: { firstName: "Bob" },
      submittedDate,
    });

    const spy = jest.spyOn(axios, "get");

    render(<PackageReview page={page} />);
    await waitFor(() => {});

    expect(spy).toHaveBeenCalled();
  });

  test("Api called, invalid submittedDate", async () => {
    window.open = jest.fn();
    mock.onGet(apiRequest).reply(200, {
      court: courtData,
      submittedDate: "123",
    });

    const spy = jest.spyOn(axios, "get");

    render(<PackageReview page={page} />);
    await waitFor(() => {});

    expect(spy).toHaveBeenCalled();
  });

  test("Api called, missing response data", async () => {
    window.open = jest.fn();
    mock.onGet(apiRequest).reply(200);

    const spy = jest.spyOn(axios, "get");

    render(<PackageReview page={page} />);
    await waitFor(() => {});

    expect(spy).toHaveBeenCalled();
  });

  test("View Submission Sheet (on click) - successful", async () => {
    const blob = new Blob(["foo", "bar"]);

    global.URL.createObjectURL = jest.fn();
    global.URL.createObjectURL.mockReturnValueOnce("fileurl.com");

    window.open = jest.fn();
    mock.onGet(apiRequest).reply(200, {
      court: courtData,
      submittedBy,
      submittedDate,
    });
    mock
      .onGet(`/filingpackage/${packageId}/submissionSheet`)
      .reply(200, { blob });

    const { container } = render(<PackageReview page={page} />);
    await waitFor(() => {});

    fireEvent.click(getByText(container, "Print Submission Sheet"));
    await waitFor(() => {});

    expect(FileSaver.saveAs).toHaveBeenCalled();
  });

  test("View Submission Sheet (on keyDown) - successful", async () => {
    const blob = new Blob(["foo", "bar"]);

    global.URL.createObjectURL = jest.fn();
    global.URL.createObjectURL.mockReturnValueOnce("fileurl.com");

    window.open = jest.fn();
    mock.onGet(apiRequest).reply(200, {
      court: courtData,
      submittedBy,
      submittedDate,
    });
    mock
      .onGet(`/filingpackage/${packageId}/submissionSheet`)
      .reply(200, { blob });

    const { container } = render(<PackageReview page={page} />);
    await waitFor(() => {});

    fireEvent.keyDown(getByText(container, "Print Submission Sheet"));
    await waitFor(() => {});

    expect(FileSaver.saveAs).toHaveBeenCalled();
  });

  test("View Submission Sheet (on click) - unsuccessful", async () => {
    sessionStorage.setItem("errorUrl", "error.com");

    window.open = jest.fn();
    mock
      .onGet(apiRequest)
      .reply(200, { court: courtData, submittedBy, submittedDate });
    mock
      .onGet(`/filingpackage/${packageId}/submissionSheet`)
      .reply(400, { message: "There was an error." });

    const { container } = render(<PackageReview page={page} />);
    await waitFor(() => {});

    fireEvent.click(getByText(container, "Print Submission Sheet"));
    await waitFor(() => {});

    expect(window.open).toHaveBeenCalledWith(
      "error.com?status=400&message=There was an error.",
      "_self"
    );
  });
});
