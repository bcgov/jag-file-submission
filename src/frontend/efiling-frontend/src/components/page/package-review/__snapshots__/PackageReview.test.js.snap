// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PackageReview Component Matches the snapshot 1`] = `
<DocumentFragment>
  <main>
    <header
      class="bcgov-header"
    >
      <nav
        aria-label="Header"
        class="container-fluid navbar navbar-expand-lg navbar-dark"
      >
        <div
          aria-labelledby="title"
          class="navbar-brand bcgov-pointer"
        />
        <img
          alt="B.C. Government Logo"
          class="img-fluid d-none d-md-block bcgov-pointer"
          height="44"
          src="undefined/images/bcid-logo-rev-en.svg"
          width="181"
        />
        <img
          alt="B.C. Government Logo"
          class="img-fluid d-md-none bcgov-pointer"
          height="44"
          src="undefined/images/bcid-symbol-rev.svg"
          width="64"
        />
        <div
          aria-labelledby="title"
          class="bcgov-pointer navbar-brand nav-item nav-link"
        />
        <div
          class="navbar-brand"
          id="title"
        >
          eFiling Frontend
        </div>
      </nav>
    </header>
    <div
      class="page"
    >
      <div
        class="content col-md-8"
      >
        <h1>
          View Recently Submitted Package # 1
        </h1>
        <br />
        <div
          class="row"
        >
          <h2
            class="col-sm-6"
          >
            Package Details
          </h2>
          <div
            class="col-sm-6 text-sm-right mt-3 mt-sm-0"
          >
            <span
              class="file-href"
              role="button"
              tabindex="0"
            >
              Print Submission Sheet
            </span>
            <svg
              class="align-icon"
              color="#7F7F7F"
              fill="currentColor"
              height="24"
              stroke="currentColor"
              stroke-width="0"
              style="color: rgb(127, 127, 127);"
              viewBox="0 0 16 16"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M10.5 8a2.5 2.5 0 11-5 0 2.5 2.5 0 015 0z"
              />
              <path
                clip-rule="evenodd"
                d="M0 8s3-5.5 8-5.5S16 8 16 8s-3 5.5-8 5.5S0 8 0 8zm8 3.5a3.5 3.5 0 100-7 3.5 3.5 0 000 7z"
                fill-rule="evenodd"
              />
            </svg>
          </div>
        </div>
        <br />
        <div
          class="row"
        >
          <div
            class="col-sm-12 col-lg-6"
          >
            <div
              class="bcgov-table "
            >
              <b />
              <div
                class="bcgov-table-body"
              >
                <div
                  class="bcgov-row "
                >
                  <div
                    class="bcgov-fill-width"
                  >
                    Submitted By:
                  </div>
                  <div
                    class="bcgov-table-value "
                  >
                    <b>
                      Han Solo
                    </b>
                  </div>
                </div>
                <div
                  class="bcgov-row "
                >
                  <div
                    class="bcgov-fill-width"
                  >
                    Submitted Date:
                  </div>
                  <div
                    class="bcgov-table-value "
                  >
                    <b>
                      14-Jan-2021 10:57
                    </b>
                  </div>
                </div>
                <div
                  class="bcgov-row "
                >
                  <div
                    class="bcgov-fill-width"
                  >
                    Submitted To:
                  </div>
                  <div
                    class="bcgov-table-value "
                  >
                    <b>
                      Kelowna Law Courts
                    </b>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="col-sm-12 col-lg-6"
          >
            <div
              class="bcgov-table "
            >
              <b />
              <div
                class="bcgov-table-body"
              >
                <div
                  class="bcgov-row "
                >
                  <div
                    class="bcgov-fill-width"
                  >
                    Package Number:
                  </div>
                  <div
                    class="bcgov-table-value "
                  >
                    <b>
                      1
                    </b>
                  </div>
                </div>
                <div
                  class="bcgov-row "
                >
                  <div
                    class="bcgov-fill-width"
                  >
                    Court File Number:
                  </div>
                  <div
                    class="bcgov-table-value "
                  >
                    <b>
                      Court file number
                    </b>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <br />
        <nav
          class="nav nav-tabs"
          role="tablist"
        >
          <a
            aria-controls="uncontrolled-tab-tabpane-documents"
            aria-selected="true"
            class="nav-item nav-link active"
            data-rb-event-key="documents"
            href="#"
            id="uncontrolled-tab-tab-documents"
            role="tab"
          >
            Documents
          </a>
          <a
            aria-controls="uncontrolled-tab-tabpane-comments"
            aria-selected="false"
            class="nav-item nav-link"
            data-rb-event-key="comments"
            href="#"
            id="uncontrolled-tab-tab-comments"
            role="tab"
            tabindex="-1"
          >
            Filing Comments
          </a>
        </nav>
        <div
          class="tab-content"
        >
          <div
            aria-hidden="false"
            aria-labelledby="uncontrolled-tab-tab-documents"
            class="fade tab-pane active show"
            id="uncontrolled-tab-tabpane-documents"
            role="tabpanel"
          >
            <br />
            Documents coming ...
          </div>
          <div
            aria-hidden="true"
            aria-labelledby="uncontrolled-tab-tab-comments"
            class="fade tab-pane"
            id="uncontrolled-tab-tabpane-comments"
            role="tabpanel"
          >
            <br />
            <h4>
              Filing Comments
            </h4>
            <div
              class="tabContent"
              id="filingComments"
            >
              Lorem ipsum dolor sit amet.&lt;script&gt;alert('Hi');&lt;/script&gt;

Duis aute irure dolor.
            </div>
          </div>
        </div>
        <br />
        <section
          class="buttons pt-2"
        >
          <button
            class="bcgov-button normal-white btn"
            data-test-id=""
            type="button"
          >
            Cancel and Return to Parent App
          </button>
        </section>
      </div>
    </div>
    <footer
      class="bcgov-footer"
    >
      <nav
        aria-label="Footer"
        class="navbar navbar-expand-sm navbar-dark justify-content-end"
      >
        <button
          class="navbar-toggler"
          data-target="#footerBar"
          data-toggle="collapse"
          type="button"
        >
          <span
            class="navbar-toggler-icon"
          />
        </button>
        <div
          class="collapse navbar-collapse flex-grow-0"
          id="footerBar"
        >
          <ul
            class="navbar-nav text-right"
          >
            <li
              class="nav-item m-1"
            >
              <a
                class="nav-link"
                href="https://www2.gov.bc.ca"
                rel="noopener noreferrer"
                target="_blank"
              >
                BC Government
              </a>
            </li>
            <li
              class="nav-item m-1"
            >
              <a
                class="nav-link"
                href="https://www2.gov.bc.ca/gov/content/home/disclaimer"
                rel="noopener noreferrer"
                target="_blank"
              >
                Disclaimer
              </a>
            </li>
            <li
              class="nav-item m-1"
            >
              <a
                class="nav-link"
                href="https://www2.gov.bc.ca/gov/content/home/privacy"
                rel="noopener noreferrer"
                target="_blank"
              >
                Privacy
              </a>
            </li>
            <li
              class="nav-item m-1"
            >
              <a
                class="nav-link"
                href="https://www2.gov.bc.ca/gov/content/home/accessibility"
                rel="noopener noreferrer"
                target="_blank"
              >
                Accessibility
              </a>
            </li>
            <li
              class="nav-item m-1"
            >
              <a
                class="nav-link"
                href="https://www2.gov.bc.ca/gov/content/home/copyright"
                rel="noopener noreferrer"
                target="_blank"
              >
                Copyright
              </a>
            </li>
          </ul>
        </div>
      </nav>
    </footer>
  </main>
</DocumentFragment>
`;
